dojo/tools/blackduck/importer.py

```diff
@@ -0,0 +1,78 @@
+from abc import ABC, abstractmethod
+from collections import defaultdict
+from pathlib import Path
+from typing import Iterable, Mapping, Set, Tuple
+from dojo.tools.blackduck.model import BlackduckFinding
+import csv
+import io
+import zipfile
+
+
+class Importer(ABC):
+    @abstractmethod
+    def parse_findings(self, report: Path) -> Iterable[BlackduckFinding]:
+        while False:
+            yield None
+
+
+class BlackduckImporter(Importer):
+    def parse_findings(self, report: Path) -> Iterable[BlackduckFinding]:
+        files = dict()
+        security_issues = dict()
+        try:
+            if zipfile.is_zipfile(report):
```

Comment:
```suggestion
            if zipfile.is_zipfile(string(report)):
```

The string is no longer needed when 3.6 is the base Python version.